pkgs/by-name/lo/lovr-playspace/package.nix

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: 2025 Red_M <nixpkgs-xr@red-m.net>
+#
+# SPDX-License-Identifier: MIT
+
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  symlinkJoin,
+  writeTextFile,
+  bash,
+  lovr,
+
+  xrSources,
+}:
+
+symlinkJoin rec {
+  inherit (xrSources.lovr-playspace)
+    pname
+    version
+    ;
+  paths = [
+    lovr
+    (stdenv.mkDerivation {
+      inherit (xrSources.lovr-playspace)
+        src
+        version
+        ;
+      pname = "${pname}-unwrapped";
+
+      dontUseCmakeConfigure = true;
+
+      installPhase = ''
+        runHook preInstall
+        mkdir -p $out/lovr-playspace $out/lovr-playspace/json
+        cp -r ./*.lua $out/lovr-playspace/
+        cp -r ./json/*.lua $out/lovr-playspace/json/
+        runHook postInstall
+      '';
+    })
+    (writeTextFile {
+      name = "lovr-playspace-script";
+      executable = true;
+      destination = "/bin/lovr-playspace";
+      text = ''
+        #!${bash}/bin/bash
+        ${lovr}/bin/lovr ${builtins.elemAt paths 1}/lovr-playspace
+      '';
+    })
```

Comment:
I think this should just be the main derivation where `lovr-playspace-script` is just a wrapper produced using `makeWrapper`. If you don't know what I mean, I can rewrite this derivation

Reply:
I'd prefer to keep this separated for if other VR overlays (or entire games) so they can use this.

I don't think `makeWrapper` works here, `lovr` has to be told the directory you wish to run, its not an individual file. Happy for you to change it if it does work that way.

Reply:
I might have just copy-pasted those `rec` parts from other working builds I had to use as a template to start from, I think these don't need them so I'll just remove them.